internal/storage/service_test.go

```diff
@@ -363,36 +275,39 @@ func TestServiceSubmit(t *testing.T) {
 		})
 		assert.Assert(t, ret == nil)
 		assert.Equal(t, err.(*goa.ServiceError).Name, "not_valid")
-		assert.ErrorContains(t, err, "cannot persist package")
+		assert.ErrorContains(t, err, "cannot sign URL")
 	})
 
 	t.Run("Returns signed URL", func(t *testing.T) {
 		t.Parallel()
 
-		// Fake internal location, using fileblob because it can generate signed URLs.
-		furl, err := url.Parse("file:///tmp/dir")
-		assert.NilError(t, err)
-		b, err := fileblob.OpenBucket("/tmp", &fileblob.Options{URLSigner: fileblob.NewURLSignerHMAC(furl, []byte("1234"))})
-		assert.NilError(t, err)
+		td := tfs.NewDir(t, "enduro-service-test")
 
-		AIPID := uuid.MustParse("5ab42bc3-acc2-420b-bbd0-76efdef94828")
-		attrs := &setUpAttrs{
-			internalLocationFactory: fakeInternalLocationFactory(t, b),
+		attrs := setUpAttrs{
+			config: ref.New(storage.Config{
```

Comment:
You can just use the & operator in Go. `ref.New` was created for literals:

    config := &"foobar" // won't compile because the string is not stored into a var, i.e. no memory address